addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -172,6 +172,24 @@ $-forum-sidebar-width: map-get($container-max-widths, sm) / 2;
         border: 0;
     }
 
+    // Prevent background display inside the wysiwyg loader
+    .bg-none {
```

Comment:
it's a not a good practice to have generic name like that in a module other than web, can you just add with forum classes ?